src/util/color/predefinedcolorpalettes.cpp

```diff
@@ -241,367 +241,333 @@ constexpr mixxx::RgbColor kTritKeyColor10(0x666600);
+                    {
+                            kMIKKeyColor1,
+                            kMIKKeyColor2,
+                            kMIKKeyColor3,
+                            kMIKKeyColor4,
+                            kMIKKeyColor5,
+                            kMIKKeyColor6,
+                            kMIKKeyColor7,
+                            kMIKKeyColor8,
+                            kMIKKeyColor9,
+                            kMIKKeyColor10,
+                            kMIKKeyColor11,
+                            kMIKKeyColor12,
+                    }},
+            .protKeyColorPalette{
+                    QStringLiteral(QT_TRANSLATE_NOOP("PredefinedColorPalettes",
+                            "Protanopia / Protanomaly Key Colors")),
+                    {
+                            kProtKeyColor1,
+                            kProtKeyColor2,
+                            kProtKeyColor3,
+                            kProtKeyColor4,
+                            kProtKeyColor5,
+                            kProtKeyColor6,
+                            kProtKeyColor7,
+                            kProtKeyColor8,
+                            kProtKeyColor9,
+                            kProtKeyColor10,
+                            kProtKeyColor11,
+                            kProtKeyColor12,
+                    }},
+            .deutKeyColorPalette{
+
+                    QStringLiteral(QT_TRANSLATE_NOOP("PredefinedColorPalettes",
+                            "Deuteranopia / Deuteranomaly Key Colors")),
+                    {
+                            kDeutKeyColor1,
+                            kDeutKeyColor2,
+                            kDeutKeyColor3,
+                            kDeutKeyColor4,
+                            kDeutKeyColor5,
+                            kDeutKeyColor6,
+                            kDeutKeyColor7,
+                            kDeutKeyColor8,
+                            kDeutKeyColor9,
+                            kDeutKeyColor10,
+                            kDeutKeyColor11,
+                            kDeutKeyColor12,
+                    }},
+            .tritKeyColorPalette{
+                    QStringLiteral(QT_TRANSLATE_NOOP("PredefinedColorPalettes",
+                            "Tritanopia / Tritanomaly Key Colors")),
+                    {
+                            kTritKeyColor1,
+                            kTritKeyColor2,
+                            kTritKeyColor3,
+                            kTritKeyColor4,
+                            kTritKeyColor5,
+                            kTritKeyColor6,
+                            kTritKeyColor7,
+                            kTritKeyColor8,
+                            kTritKeyColor9,
+                            kTritKeyColor10,
+                            kTritKeyColor11,
+                            kTritKeyColor12,
+                    }},
+            .defaultHotcueColorPalette = kPalettes.mixxxHotcueColorPalette,
```

Comment:
This is a deep copy. Can we avoid it?

Reply:
we can simply store a const ref because we know the object outlives the reference.